pkg/api/connection/mechanisms/sriovuserspace/constants.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020 Intel Corporation. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sriovuserspace
+
+const (
+	// MECHANISM string
+	MECHANISM = "SRIOV_USERSPACE"
```

Comment:
Question... how does the SRIOV device *look* to the consumer in the userspace case?  My recollection is that it shows up as a vfio device... which makes me wonder if it might make sense to refer to this as a vfio mechanism?  Just brainstorming, thoughts?

Reply:
I'm not fond of the name SRIOV_USERSPACE for the reasons you suggest. To me it's a little vague and perhaps we're splitting hairs, but I do think we could rename this to more specifically reference the actual mechanism at work be it vfio or whatever is working under the hood.

Reply:
I share this sentiment, naming this VFIO may be more apt.